aws-ebs-volume-unencrypted.yaml

```diff
@@ -0,0 +1,39 @@
+rules:
+- id: aws-ebs-volume-unencrypted
+  patterns:
+  - pattern: |
+      resource "aws_ebs_volume" $ANYTHING {
+        ...
+      }
+  - pattern-not: |
+      resource "aws_ebs_volume" $ANYTHING {
+        ...
+        encrypted = true
+        ...
+      }
+  message: The AWS EBS volume is unencrypted. The volume, the disk I/O and any derived
```

Comment:
My rule **does** use  the yaml operator `>-`:

<img width="594" alt="Screenshot 2023-02-23 at 3 36 22 PM" src="https://user-images.githubusercontent.com/5559014/221024300-cf02b15b-fd05-428a-9b43-d586669fac68.png">

I don't know why that disappears when I publish it from the semgrep app?